server/datastore/mysql/policies.go

```diff
@@ -641,7 +642,7 @@ func (ds *Datastore) CleanupPolicyMembership(ctx context.Context, now time.Time)
 				policies p
 			WHERE
 				p.updated_at >= DATE_SUB(?, INTERVAL ? SECOND) AND
-				p.created_at < p.updated_at`  // ignore newly created
+				p.created_at < p.updated_at`
```

Comment:
gofmt complaining on macOS and Linux in different ways, so I changed it so that it doesn't complain.

Reply:
:open_mouth: